pglookout/pglookout.py

```diff
@@ -483,7 +485,10 @@ def consider_failover(self, own_state, master_node, standby_nodes):
                 self.current_master,
                 master_node,
             )
-            if self.current_master:
+            if not self._failover_on_disconnect and master_node:
+                self.stats.increase("failover_decision_on_disconnect_not_taken")
+                self.log.warning("Not considering failover, because it's not enabled by configuration")
+            elif self.current_master:
```

Comment:
About the last line of this method:
```python
self.check_replication_lag(own_state, standby_nodes)
```

This is now reached in either of the following conditions:
- `not self._failover_on_disconnect and master_node`
- `self.current_master and not master_known_to_be_gone and not config_timeout_exceeded`

Is that what we want? Or do we want to skip this check in our new case (master and flag disabled)? If so, we might want to do this instead:

```suggestion
                return
            if self.current_master:
```